web/modules/custom/modal_form_example/src/Form/Demo.php

```diff
@@ -0,0 +1,314 @@
+<?php
+
+namespace Drupal\modal_form_example\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\node\Entity\Node;
+use Drupal\file\Entity\File;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * A class to create form for creating node.
+ */
+class Demo extends FormBase {
+
+  /**
+   * To Get the Current user details.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $account;
+
+  /**
+   * To Load the file to database.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * Class constructor.
+   */
+  public function __construct(AccountInterface $account, EntityTypeManagerInterface $entity_type_manager) {
+    $this->account = $account;
+    $this->entityTypeManager = $entity_type_manager;
+
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    // Instantiates this form class.
+    return new static(
+      // Load the service required to construct this class.
+      $container->get('current_user'),
+      $container->get('entity_type.manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $step = 1;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'multi_step_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $uid = $this->account->id();
+
+    if ($uid) {
+      $user = $this->entityTypeManager->getStorage('user')->load($uid);
+      $pictureUri = $user->user_picture->entity->getFileUri();
+      $name = $user->getAccountName();
+      $picture = str_replace(
+            'public://pictures/2022-01/',
```

Comment:
This folder path can change in future, if you just want the image name use explode, or some other methods.